hieradata/common.yaml

```diff
@@ -583,6 +583,8 @@ govuk_ci::master::pipeline_jobs:
   govspeak: {}
   govuk-app-deployment: {}
   govuk-diff-pages: {}
+  govuk-dns-config:
```

Comment:
As other comment, this will now also appear in the Deploy App job, and I'm not sure if we want it there. It might be worth writing in some code to remove it from this job somehow? I'm not sure how much it matters.